root/components/EntityHeader.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const {l} = require('../static/scripts/common/i18n');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+const EntityTabs = require('./EntityTabs');
+const Frag = require('./Frag');
+const SubHeader = require('./SubHeader');
+
+type Props = {|
+  +editTab?: React.Node,
+  +entity: $Subtype<CoreEntityT>,
+  +headerClass: string,
+  +heading?: string | React.Node,
+  +page: string,
+  +preHeader?: React.Node,
+  +hideEditTab?: boolean,
+  +subHeading: string,
+|};
+
+const EntityHeader = ({
+  editTab,
+  entity,
+  headerClass,
+  heading,
+  hideEditTab = false,
+  page,
+  // $FlowFixMe
+  preHeader = null,
+  subHeading,
+}: Props) => (
+  <Frag>
+    <div className={headerClass}>
+      {preHeader}
```

Comment:
I added a `preHeader` prop and moved the `TaggerLink` inclusion outside, so that the component has no entity-specific code at all.